src/nft/components/explore/Cells/Cells.tsx

```diff
@@ -27,10 +27,11 @@ const CollectionName = styled.div`
   text-overflow: ellipsis;
 `
 
-const TruncatedSubHeader = styled(ThemedText.SubHeader)`
```

Comment:
Sorry it's not clear on the ticket but this is supposed to only be smaller on mobile. Also the font color should still be textPrimary

Reply:
thats my bad, it was in the mobile explore epic so I should've noticed. It should be fixed now. I also put some repeated styles in a variable. I didn't see an existing pattern for something like that (besides sprinkles) - maybe there is a better way?

Reply:
thats my bad, it was in the mobile explore epic so I should've noticed. It should be fixed now. I also put some repeated styles in a variable. I didn't see an existing pattern for something like that (besides sprinkles) - maybe there is a better way?

Reply:
This usage is fine, you can also replaced styled.div with the common style similar to here: https://github.com/Uniswap/interface/blob/main/src/components/Card/index.tsx#L12